### ISSUE TITLE
Accordian: Should be possible to select aria-level

### ISSUE THREAD

**Issue by PasiVaisanen** [seb-oss/green#1171]
### Does the component already exist

- [X] I confirm that I have checked if the component already exists in any of the Green libraries

### For which framework/library you are requesting the component

Angular

### Description

Currently the aria-level is hardcoded to 2, it should be possible to select the aria-level depending of the content where the accordian is displayed.

### Who will benefit from this feature?

Hard to use a speech reader when we cannot select the area-level.
